src/common/ceph_context.cc

```diff
@@ -345,10 +345,10 @@ void CephContext::do_command(std::string command, cmdmap_t& cmdmap,
     if (!cmd_getval(this, cmdmap, "var", var)) {
       f->dump_string("error", "syntax error: 'perf reset <var>'");
     } else {
-     if(!_perf_counters_collection->reset(var))
+      if(!_perf_counters_collection->reset(var))
```

Comment:
@tchaikov ,

Pls, based on comment https://github.com/ceph/ceph/pull/10736#discussion_r75415073  you can see my reply to @badone , (We already had some misaligned block, so I ended up fixing some up the function in order to see this one properly). Should we not touch/fix these kind of cases? That was the only reason why it was touched.